sdk/eventhub/azure-eventhub/azure/eventhub/_connection_string_parser.py

```diff
@@ -91,7 +91,7 @@ def parse_connection_string(conn_str: str) -> "EventHubConnectionStringPropertie
     :return: A properties bag containing the parsed connection string.
     :rtype: ~azure.eventhub.EventHubConnectionStringProperties
     """
-    fully_qualified_namespace, policy, key, entity, signature = _parse_conn_str(
+    fully_qualified_namespace, policy, key, entity, signature, emulator = _parse_conn_str(
```

Comment:
where is this used?

Reply:
it isn't its to avoid an error when parse_connection_string() is called externally